src/main/java/com/entropyteam/entropay/employees/services/VacationService.java

```diff
@@ -107,4 +107,30 @@ protected Vacation toEntity(VacationDto entity) {
 
         return vacation;
     }
+    public void addVacationCredit(Employee employee, Integer totalDaysToAdd) {
```

Comment:
please remove this method and use discountVacationDebit to restore employee vacation days

Reply:
Resolved